release-notes/opendistro-for-elasticsearch-knn.release-notes-1.13.0.0.md

```diff
@@ -0,0 +1,34 @@
+## Version 1.13.0.0 Release Notes
+
+Compatible with Elasticsearch 7.10.2
+
+### Features
+
+* Support k-NN similarity functions in painless scripting ([#281](https://github.com/opendistro-for-elasticsearch/k-NN/pull/281))
+
+### Enhancements
+
+* Upgrade nmslib to v2.0.8 in JNI library ([#291](https://github.com/opendistro-for-elasticsearch/k-NN/pull/291))
```

Comment:
I think this can be removed as well, because we ended up upgrading to 2.0.11 (so users will never use 2.0.8)

Reply:
Ack